src/credits.c

```diff
@@ -2259,7 +2259,7 @@ static u8 sub_8177224(u16 species, s16 x, s16 y, u16 position)
     u8 spriteId;
     u8 spriteId2;
 
-    spriteId = sub_80C0E9C(species, x, y, position);
+    spriteId = CreateMonSpriteFromNationalDexNumber(species, x, y, position);
```

Comment:
The first arg here should be named `nationDexNum`, not `species`

Reply:
Renamed to `nationalDexNum`.